docs/mirror.rst

```diff
@@ -55,7 +55,7 @@ content conforms to. The first path component of the schema URL is ``v1``.
 +++++++++++++++++
 
 Info objects contain JSON further describing a file. The key of an info object
-is ``info/${digest_value}.${digest_type}`` where ``digest_value`` is the
+is ``info/${digest_value}.${digest_type}.json`` where ``digest_value`` is the
 hexadecimal form of the hash of the corresponding file object's content and
```

Comment:
The paragraph describing file and info objects define `digest_value` as "the hexadecimal form of **the** hash" whereas the paragraph on alias objects says "the hexadecimal form of **a** hash". I prefer the latter form, but either way they ought to be consistent.